pkg/ocm/pullsecret_auth.go

```diff
@@ -46,8 +41,10 @@ func (a authentication) AuthenticatePullSecret(ctx context.Context, pullSecret s
 
 	response, err := accessTokenAPI.TokenAuthorization().Post().Request(request).Send()
 	if err != nil {
-		// failed to send request
-		if strings.Contains(err.Error(), sendRequestError) {
+		if response.Status() >= 400 && response.Status() < 500 {
```

Comment:
Probably should validate that `response` is not nill and maybe log `response` and `err` before returning the error.

Reply:
Done